builtin/providers/aws/resource_aws_autoscaling_group_test.go

```diff
@@ -445,15 +445,6 @@ func TestAccAWSAutoScalingGroup_ALB_TargetGroups(t *testing.T) {
 						"aws_autoscaling_group.bar", "target_group_arns.#", "1"),
 				),
 			},
-
-			resource.TestStep{
-				Config: testAccAWSAutoScalingGroupConfig_ALB_TargetGroup_pre,
-				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckAWSAutoScalingGroupExists("aws_autoscaling_group.bar", &group),
-					resource.TestCheckResourceAttr(
-						"aws_autoscaling_group.bar", "target_group_arns.#", "0"),
-				),
-			},
```

Comment:
As discussed on Slack this is failing because at the core/schema level we don't handle `Optional`+`Computed` Sets very well when removing last items from them.

It's not great, but I'm ok with merging this then 👍 We'll resolve the core bug in a separate PR eventually.